docs/didkit-examples/core-functions-with-aleo.md

```diff
@@ -0,0 +1,126 @@
+---
+id: core-functions-with-aleo
+title: Core Functions with Aleo
+---
+
+## Introduction
+
+DIDKit also supports the use of Aleo accounts as verifiable credentials.
+
+This is an example shell script using all the core functions of DIDKit-CLI: key
+generation, credential/presentation issuance and verification.
+
+_Note 1: This script is meant to be in a DIDKit-CLI source directory. See the
+complete script below for setup details._
+
+_Note 2: Currently Aleo support is only available through the
+`feat/aleo-sig-pkh` branch of the ssi library. When building the DIDKit CLI the
+feature `ssi/aleosig` must also be enabled._
+
+### Start with a keypair
+
+The SSI library can generate an Aleo keypair as an example:
+
+```bash
+git clone https://github.com/spruceid/ssi
+cd ssi
+git checkout feat/aleo-sig-pkh
+cargo run --example genaleojwk --features=aleosig > aleokey.jwk
+```
+
+You can also provide the details of an existing Aleo account, although you will
+need to do some extra work for DIDKit to use it.
+
+The Aleo private JWK format used by DIDKit is non-standard. An example:
+
+```json
+{
+  "kty": "OKP",
+  "crv": "AleoTestnet1Key",
+  "x": "78_Jh_c7Fw46fX31xS9Ifdg_LeuabZ2p2aIl5fn9zw0",
+  "d": "f4a9dNLd0omQcg3SEajVHGqEqwFHDGD9yNc2xpzuiZ3sSJjIf5AnEYXWCQ"
+}
+```
+
+The format is as follows:
+
+- kty: "OKP"
+- crv: "AleoTestnet1Key"
+- x: An Aleo account address derived from the private key using Aleo Testnet1
+  parameters, as a Base64 value (without the "aleo" prefix that appears in its'
+  Base58 format)
+- d: An Aleo private key converted from Base58 (where it starts with
+  "APrivateKey1") to Base64 value
```

Comment:
JWK uses base64url, that is a variant of base64:
https://www.rfc-editor.org/rfc/rfc7515.html#page-6 JSON Web Signature (JWS) - Base64url Encoding
https://www.rfc-editor.org/rfc/rfc8037.html#section-2 CFRG ECDH and Signatures in JOSE - Key Type "OKP"
```suggestion
- x: An Aleo account address derived from the private key using Aleo Testnet1
  parameters, as a Base64Url value (without the "aleo" prefix that appears in its
  Base58 format)
- d: An Aleo private key converted from Base58 (where it starts with
  "APrivateKey1") to Base64Url value
```